File: Práctica 1/C++/practica1-prueba/zCoord.cc
#include <iostream>
#include "Coordenadas.h"
using namespace std;

int main(){
	Coordenadas cor1(3, 3);

	cout << cor1.getFila() << " " << cor1.getColumna() << endl;

	Coordenadas cor2(-1, 3);

	cout << cor2.getFila() << " " << cor2.getColumna() << endl; // -1 -1
	

	Coordenadas cor3(0, 3);

	cout << cor3.getFila() << " " << cor3.getColumna() << endl; // 0 3


	return 0;
}

File: Práctica 1/Java/practica1-prueba/zCoord.java
public class zCoord {
	public static void main(String [] args) {
		Coordenadas cor1 = new Coordenadas(3, 3);
		System.out.println(cor1.getFila() + " " + cor1.getColumna());
		
		Coordenadas cor2 = new Coordenadas(-1, 3);
		System.out.println(cor2.getFila() + " " + cor2.getColumna());
		
		Coordenadas cor3 = new Coordenadas(0, 3);
		System.out.println(cor3.getFila() + " " + cor3.getColumna());
		
	}
}
